test/testfixture/customize_entity_funcs.go

```diff
@@ -62,6 +62,88 @@ func SetTopologies(topologies ...link.Topology) CustomizeWorkItemLinkTypeFunc {
 	}
 }
 
+// Link is a helper struct to construct a link from two work item titles
+// (instead of the work item IDs). It is mainly used by the `L`, `LinkChain` and
+// `BuildLinks` functions.
+type Link struct {
+	SourceTitle   string
+	TargetTitle   string
+	LinkTypeTitle string
+}
+
+// L constructs and individual link from the source and target work item titles
+// with an optional link type name.
+func L(SourceTitle string, TargetTitle string, LinkTypeTitle ...string) Link {
```

Comment:
I don't think we need to export this.

Reply:
@pranavgore09 I want to export this because it is meant to speed up individual linking (without chains):

```go
fxt := tf.NewTestFixture(t, s.DB,
	tf.WorkItemLinkTypes(1, tf.SetTopologies(link.TopologyDependency)),
	tf.WorkItems(7, tf.SetWorkItemTitles("A", "B", "C", "D", "E", "F", "G")),
	tf.WorkItemLinksCustom(5, tf.BuildLinks(tf.L("A", "B"), tf.L("C", "G"))),
)
```

The short name was intentionally picked to be able to create small topologies very fast.

Reply:
hmm, okay I didn't see that
thanks for example